### PR TITLE
[5.5] fix(13909): fixed update_at column is ambiguous-column by qualifying …

### PR DESCRIPTION
**Pull request by nickshatilo**
Fixes the #13909 issue for the 5.5 version.

### CODE REVIEW

**Comment by taylorotwell** [comment]
Tests break on PHP 7.2.

File: src/Illuminate/Database/Eloquent/Builder.php [hunk 1]
```diff
@@ -822,10 +822,15 @@ protected function addUpdatedAtColumn(array $values)
             return $values;
         }
 
-        return Arr::add(
-            $values, $this->model->getUpdatedAtColumn(),
-            $this->model->freshTimestampString()
-        );
+        if (count($this->getQuery()->joins) > 0) {
```
Comment (mfn) [inline src/Illuminate/Database/Eloquent/Builder.php]: Like this is the PHP 7.2 break in case `joins` is e.g. null?

Comment (nickshatilo) [reply src/Illuminate/Database/Eloquent/Builder.php]: @mfn Yep. Fixed in another PR

Comment (nickshatilo) [reply src/Illuminate/Database/Eloquent/Builder.php]: @mfn https://github.com/laravel/framework/pull/26022

**Pull request closed without merging**